applications/prisme/src/app/home/home-footer/home-footer.component.ts

```diff
@@ -10,7 +10,7 @@ export class HomeFooterComponent {
 	figma = {
 		title: 'Une maquette a faire ?',
 		description: 'Découvrez notre librairie figma',
-		link: 'https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FFvgujSMHAxgXwPU05xent4AJ%2FLucca-Front-Library%3Fnode-id%3D1666%253A90',
+		link: 'https://www.figma.com/file/sO3msermeMuyFvUAcXrbn5/Main-Components-2.0?node-id=0%3A1',
```

Comment:
We're publishing the link on a public site but only Figma users who have been granted access to the file can visit it.